core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -48,166 +48,175 @@
     </div>
     <div ng-cloak class="navbar-header float-right">
       <ul class="nav oppia-navbar-nav oppia-navbar-profile-admin">
-        <ul class="nav oppia-navbar-tabs-admin oppia-admin-page-dropdown" uib-dropdown>
-          <li uib-dropdown-toggle class="nav-item oppia-clickable-navbar-element"
-              ng-mouseover="$ctrl.onMouseoverDropdownIconOrMenu($event)"
-              ng-mouseleave="$ctrl.onMouseoutDropdownIconOrMenu($event)">
-            <a class="nav-link oppia-navbar-tab"
-               uib-tooltip="Menu"
-               tooltip-placement="bottom">
-              <i class="fa fa-bars oppia-admin-page-dropdown-icon"></i>
-            </a>
-          </li>
-          <ul class="uib-dropdown-menu dropdown-menu oppia-admin-page-dropdown-menu"
-              ng-mouseover="$ctrl.onMouseoverDropdownIconOrMenu($event)"
-              ng-mouseleave="$ctrl.onMouseoutDropdownIconOrMenu($event)"
-              ng-show="$ctrl.dropdownMenuisActive"
-              role="menu">
-            <li ng-repeat="(tabName, url) in $ctrl.ADMIN_TAB_URLS">
-              <a class="dropdown-item nav-link oppia-navbar-tab"
-                 style = "color: #00376d"
-                 ng-href="<[url]>">
-                 <[tabName]>
+        <li>
```

Comment:
I'm not sure about this change (from line 51 - 219), can you please explain?

Reply:
According to the Lighthouse Aduits, Lists should only contain <li> elements and script supporting elements (<script> and <template>)
https://web.dev/list/?utm_source=lighthouse&utm_medium=devtool

In this case, nested `<ul>` elements need to be inside a `<li>` element. 

Does that make sense? I will update my PR with a high level overview of the fixes I made soon

Reply:
Ah, I see! Thanks for the info and the ref link! 
Can you please share the screenshot of the UI after this change?

Reply:
Yes, Just updated the PR description